lib/bson/parser/utils.js

```diff
@@ -5,7 +5,7 @@
  * @param {Function} fn The function to stringify
  */
 function normalizedFunctionString(fn) {
-  return fn.toString().replace('function(', 'function (');
+  return fn.toString().replace(/function(.*)\(/, 'function (');
```

Comment:
1. This is a good fix
2. I think we should submit this as a separate change. It kind of gets buried here.